docs/src/pages/layout/grid/grid.md

```diff
@@ -20,6 +20,8 @@ The grid system is implemented with the `Grid` component:
 - Items have padding to create the spacing between individual items.
 - There are five grid breakpoints: xs, sm, md, lg, and xl.
 
+If you are **new to or unfamiliar with flexbox**, we encourage you to read this [CSS-Tricks flexbox](https://css-tricks.com/snippets/css/a-guide-to-flexbox/) guide.
```

Comment:
The top referral of css-tricks is Bootstrap. The flexbox grid is one of their most views blog post. People need this content:
![Capture d’écran 2019-03-23 à 01 25 11](https://user-images.githubusercontent.com/3165635/54859194-9a955380-4d0a-11e9-8258-08327d3935d0.png)

Reply:
Probably my most visited design resource (and top google result for flexbox), although sadly, after the recent redesign, the diagrams  are not as clear as they used to be.